文档/zh_cn/教程/How to support new model

```diff
@@ -0,0 +1,86 @@
+## 如何支持新的模型
```

Comment:
could put this file into `docs/zh_cn/tutorials/how_to_support_new_models.md` ? It will be aligned with the english version.

Reply:
@matrixgame2018  Hi, actually you don't have to fix in another new PR. Just work on this PR is fine.